httpcodec/http.pb.go

```diff
@@ -0,0 +1,275 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Can you update the `Makefile` so this is compiled into a `types` directory?

It'll:

1. Make it easier to import
2. Make it self explanatory (e.g. `types.HTTPResponse`)
3. Follow the same patterns as poktroll and the cosmos SDK